publish/package.json

```diff
@@ -35,6 +35,7 @@
     "vue": "^2.6.11",
     "fecha": "^4.2.1",
     "imask": "^6.0.7",
+    "lodash.orderby": "^4.6.0",
```

Comment:
this changes already on another PR, isn't it? @adzharamrullah 🤔

Reply:
done i fixed it